config/manager/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/rhdh-community/operator
-  newTag: 0.2.3
```

Comment:
This is the default version of the operator in this branch and should not be updated.

/close